setup/index.js

```diff
@@ -38,6 +38,11 @@ deleteFile('index.js');
 deleteFile('../App.js');
 deleteDirectory('.');
 
+console.log('\n📝 Committing project...');
+execSync('git add . && git commit -m "Initialize new React Native project."', {
```

Comment:
Prefer `"first commit"`  😆

Reply:
how about just "first"

Reply:
I was actually going to do something creative here, but figured it was better to stay generic.